cdc/processor/sinkmanager/manager.go

```diff
@@ -86,8 +86,12 @@ type SinkManager struct {
 	sourceManager *sourcemanager.SourceManager
```

Comment:
There are 2 constant variable `maxRetryDuration` and `errGCInterval` not used, can you remove them ?